### PR TITLE
Displaying issue status to student open/closed

### PR DESCRIPTION
**Pull request by shresta4**
Displayed whether the issue is open/closed to the student. 
Below, issue status is "open". 

image

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -4920,8 +4920,8 @@
             "from": "git://github.com/mathjax/MathJax-src.git",
             "requires": {
                 "esm": "^3.2.25",
-                "mj-context-menu": "^0.2.0",
-                "speech-rule-engine": "^3.0.0-beta.6"
+                "mj-context-menu": "^0.2.2",
+                "speech-rule-engine": "^3.0.0-beta.8"
```
Comment (mwest1066) [inline package-lock.json]: Is this change needed in this PR?

File: pages/partials/question.ejs [hunk 2]
```diff
@@ -41,19 +41,33 @@
...
+        <% if (issue.open) { %>
+          <span type="button" class="badge badge-danger" data-toggle="collapse" href="#">
+            Open
+          </span>
+          <% } else { %>
+            <span type="button" class="badge badge-success" data-toggle="collapse" href="#">
+              Closed
+            </span>
+          <% } %>
+      </p>
```
Comment (mwest1066) [inline pages/partials/question.ejs]: Can we put the issue status in the table above? That table is used for standard issue data, and I think "status" is like this. It would also avoid the need to change around the authorization checking.

File: pages/partials/question.ejs [hunk 3]
```diff
@@ -41,19 +41,33 @@
...
         <button type="button" class="btn btn-xs btn-secondary" data-toggle="collapse" href="#issue-system-data-<%= iIssue %>" aria-expanded="false" aria-controls="#issue-system-data-<%= iIssue %>">
           Show/hide
         </button>
       </p>
+      <% } %>
+      <p><strong>Issue status:</strong>
+        <% if (issue.open) { %>
+          <span type="button" class="badge badge-danger" data-toggle="collapse" href="#">
+            Open
+          </span>
```
Comment (mwest1066) [inline pages/partials/question.ejs]: Why are these `type="button"`? They don't seem to anything when clicked, if I'm reading the code correctly?

**Comment by dkim221** [comment]
I'll work on this project.

**Comment by mwest1066** [comment]
Re-PRed as #2203
